docs/user-guide.md

```diff
@@ -0,0 +1,76 @@
+# User Guide
+
+## Workflows
+
+## Verify Kani harnesses
+
+### View Kani harnesses
+
+As soon as the Rust package containing the harnesses is opened using the Kani extension in a VSCode instance, you should see the Kani harnesses loaded as regular unit tests in the testing panel on the left border of the VSCode window.
+
+This is how the VSCode window looks like when you click on the panel:
+
+![Verify Proofs](../resources/screenshots/first.png)
+
+### Run Kani harnesses
+
+You can then run your harnesses using the harness tree view by clicking the play button beside the harness that was automatically picked up by the Kani VSCode Extension.
+Once you run the harness using the extension, you are shown an error message if the verification fails.
+You are then presented with two options:
+ 1. Generate the report for the harness.
+ 2. Run concrete playback to generate unit tests.
```

Comment:
The order of these two options is different from the order of paragraphs that explain them. It would be better to make them consistent: first introduce the trace report, and then the concrete playback.

Also, it would be better to have a sentence after these two options to tell the users that we are going to introduce them separately in the following paragraphs. Or using a link to jump to the corresponding paragraph.

Reply:
Would it be okay to present the two options in the reverse then? We want to encourage users to use concrete playback first, before viewing the trace so I can reverse them while introducing them.

Reply:
Or you can keep the order of the options and add a sentence to tell readers that we will explain the concrete playback first.

Reply:
I've added an explanation, and done what you've suggested.